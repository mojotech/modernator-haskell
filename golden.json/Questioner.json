{
    "seed": -6.11273202426954117e18,
    "samples": [
        {
            "questionerId": -19.0,
            "name": "",
            "sessionId": 9.0
        },
        {
            "questionerId": -5.0,
            "name": null,
            "sessionId": 6.0
        },
        {
            "questionerId": 27.0,
            "name": "\"R>omíøÔU[u÷",
            "sessionId": 15.0
        },
        {
            "questionerId": -24.0,
            "name": "¼êÇZ\u0001\u0004\u0007L\u001d'ÄH\u0010m",
            "sessionId": -10.0
        },
        {
            "questionerId": 20.0,
            "name": "cq\\%²\u0003\u000bV5w&GÒµ",
            "sessionId": -2.0
        },
        {
            "questionerId": 25.0,
            "name": "\u0016\u0008p^\u0013\u0011(á\u000fRx=\u000bR0\u000bv+",
            "sessionId": 15.0
        },
        {
            "questionerId": -17.0,
            "name": "_Ï|v\u0012°PèÅ\u0008ª9w''\u001cg",
            "sessionId": -14.0
        },
        {
            "questionerId": -9.0,
            "name": "r",
            "sessionId": 5.0
        },
        {
            "questionerId": 10.0,
            "name": "P\u0010\u0004æ",
            "sessionId": 16.0
        },
        {
            "questionerId": -30.0,
            "name": "\u0011væ,r@\u001f",
            "sessionId": 6.0
        },
        {
            "questionerId": 9.0,
            "name": "f0\u0018Ç­µ|Wàj®ÉRN;\u0014\u0004â*\u0018\u0001Ë+hp0",
            "sessionId": -16.0
        },
        {
            "questionerId": 8.0,
            "name": "\u0006\u0011¾JÃ°\u0007ð\u000fn\u001ek~µ<h)\u000e}",
            "sessionId": 1.0
        },
        {
            "questionerId": 25.0,
            "name": "¡6 f ¡",
            "sessionId": -22.0
        },
        {
            "questionerId": 14.0,
            "name": "\u0014",
            "sessionId": 27.0
        },
        {
            "questionerId": -1.0,
            "name": "\u0003\u001aòWîK \u0002äCh©\u0007cöÉ\u0007v\u0001o+",
            "sessionId": 23.0
        },
        {
            "questionerId": 17.0,
            "name": "\u0013~&\u0008",
            "sessionId": 1.0
        },
        {
            "questionerId": -22.0,
            "name": "\\\u001eH\t:~\u0012jz ¦ZP\u0019yD{1|;",
            "sessionId": 9.0
        },
        {
            "questionerId": 7.0,
            "name": null,
            "sessionId": -18.0
        },
        {
            "questionerId": -10.0,
            "name": "\u001f\u000bLLe¡{dã¥\u0008«à1I",
            "sessionId": -7.0
        },
        {
            "questionerId": 15.0,
            "name": "hYho\u0018F$\u0000¡À9â\u00146\u00199pDe?tW",
            "sessionId": 27.0
        },
        {
            "questionerId": -26.0,
            "name": null,
            "sessionId": 10.0
        },
        {
            "questionerId": 27.0,
            "name": "C{\u0006©m\u0018>ä+\u0018\r¿¾R\u0007?û\u001b\"\u0003V\u000e_*r",
            "sessionId": 11.0
        },
        {
            "questionerId": 6.0,
            "name": "à*r",
            "sessionId": -6.0
        },
        {
            "questionerId": 0.0,
            "name": "\u0002@\u0005 uêfy\u0002\u0004{ ¶XGd\u000cMFît7KjÕ[",
            "sessionId": 6.0
        },
        {
            "questionerId": -16.0,
            "name": null,
            "sessionId": 16.0
        },
        {
            "questionerId": -20.0,
            "name": "ñ\u001d&.Ç¥B.)}.»D½\u0012z\u0015«&s",
            "sessionId": 4.0
        },
        {
            "questionerId": -16.0,
            "name": "F\u001aß",
            "sessionId": 11.0
        },
        {
            "questionerId": -26.0,
            "name": null,
            "sessionId": -25.0
        },
        {
            "questionerId": -11.0,
            "name": "Ý(Ó/\u0011~",
            "sessionId": 17.0
        },
        {
            "questionerId": 1.0,
            "name": "SMD\u0014\u0015dä\u0006\u00109y",
            "sessionId": 28.0
        },
        {
            "questionerId": 4.0,
            "name": "y\\älo HYÔ\u0007H[sJV­¦\u0004",
            "sessionId": 10.0
        },
        {
            "questionerId": -22.0,
            "name": "D\u001d\u001c°C\u000e^¿\u0006O0\u001bjÓEe\u000c!L}Õ?\u0015KP",
            "sessionId": 23.0
        },
        {
            "questionerId": 22.0,
            "name": "Öµt\u001cF\u0019dÏæ#ÙV##d:z÷",
            "sessionId": -24.0
        },
        {
            "questionerId": -13.0,
            "name": "ËÆ\u0012T'|Gp\u001a\u0018?_\u0007\u001eÎNÃ¬Þ\rM$J",
            "sessionId": 25.0
        },
        {
            "questionerId": 23.0,
            "name": "-\r1>i\tÃ\"}³)he¥,,|>",
            "sessionId": 9.0
        },
        {
            "questionerId": -15.0,
            "name": ",8÷Aï|\u0003f\u0011.H\r#\"\u0003w\r/V\r",
            "sessionId": -8.0
        },
        {
            "questionerId": 21.0,
            "name": "\u0007+ôO2\u0013ð",
            "sessionId": -8.0
        },
        {
            "questionerId": 6.0,
            "name": "ëF38X\u0013z%n!7]\u0004",
            "sessionId": 1.0
        },
        {
            "questionerId": -27.0,
            "name": "Vvu\u001fd0:\u0011A¤3",
            "sessionId": -14.0
        },
        {
            "questionerId": 3.0,
            "name": "A}:",
            "sessionId": 29.0
        },
        {
            "questionerId": 15.0,
            "name": null,
            "sessionId": -7.0
        },
        {
            "questionerId": -13.0,
            "name": "¾dBoUôRm\u0017©¤ò\u0016",
            "sessionId": -11.0
        },
        {
            "questionerId": 0.0,
            "name": null,
            "sessionId": -7.0
        },
        {
            "questionerId": 14.0,
            "name": "Y\u0016?8\u0007{\u0013C1#Y¨Ê ó",
            "sessionId": -29.0
        },
        {
            "questionerId": -25.0,
            "name": "ACåk\u0003X®/²7up¼,t\u00188ê5*»\u001a",
            "sessionId": 15.0
        },
        {
            "questionerId": -10.0,
            "name": null,
            "sessionId": -23.0
        },
        {
            "questionerId": 24.0,
            "name": "\u000b1u;uò%à~=/\u0002;I.\u000e",
            "sessionId": 22.0
        },
        {
            "questionerId": 6.0,
            "name": "J+\u0010gÞ\u0000ÒQD",
            "sessionId": -5.0
        },
        {
            "questionerId": -29.0,
            "name": null,
            "sessionId": 3.0
        },
        {
            "questionerId": -7.0,
            "name": "m;cÈ9­Í~Zn$'¡Jt\u0018dq\u0015N",
            "sessionId": -25.0
        },
        {
            "questionerId": -3.0,
            "name": "á?Y¬\u0015íhqsg\r\"\u0011/¢:\r\u001eNÝr\u000fj,xY,e",
            "sessionId": 1.0
        },
        {
            "questionerId": -25.0,
            "name": "N(",
            "sessionId": -5.0
        },
        {
            "questionerId": 23.0,
            "name": "&,HÞñ\u0006\u0002ae",
            "sessionId": 1.0
        },
        {
            "questionerId": 23.0,
            "name": "\\J\u0012¡`N\u001d ^",
            "sessionId": 10.0
        },
        {
            "questionerId": 21.0,
            "name": "5k3\u001fD?\u001e\u0013#(Õ\n\u0006\u0010\u0004Ë\u0004",
            "sessionId": -15.0
        },
        {
            "questionerId": 20.0,
            "name": null,
            "sessionId": -3.0
        },
        {
            "questionerId": 10.0,
            "name": "ÊÒá\u000eõ|ä[^\u001fh\u001fFD9¦\u001alG\u0018¹rç@\t*\u001d\u0015`",
            "sessionId": -1.0
        },
        {
            "questionerId": -29.0,
            "name": "53\u001eK]µw\u0000e\n\u001f=`\u001b\u0011Xi",
            "sessionId": -26.0
        },
        {
            "questionerId": 7.0,
            "name": null,
            "sessionId": 21.0
        },
        {
            "questionerId": 24.0,
            "name": "Z@\u00162C",
            "sessionId": 11.0
        },
        {
            "questionerId": -10.0,
            "name": null,
            "sessionId": -13.0
        },
        {
            "questionerId": -6.0,
            "name": null,
            "sessionId": 7.0
        },
        {
            "questionerId": -11.0,
            "name": "gWÌbvD|Ì<lRv´ú(\u000eFQ ¤\t",
            "sessionId": -16.0
        },
        {
            "questionerId": 19.0,
            "name": "+ñ£/\u0013(ø(c~\u0002nH0\u001fO\u0000\u0013æ)e[\u001c´",
            "sessionId": -19.0
        },
        {
            "questionerId": -5.0,
            "name": "´\u0008*\u001fR\u0015*ÅXSMeioxD1%÷\u0006vü",
            "sessionId": 4.0
        },
        {
            "questionerId": 3.0,
            "name": "N",
            "sessionId": 24.0
        },
        {
            "questionerId": -16.0,
            "name": "\u0019c½+",
            "sessionId": 18.0
        },
        {
            "questionerId": 28.0,
            "name": null,
            "sessionId": -29.0
        },
        {
            "questionerId": -29.0,
            "name": "²^Ð<)\u0000Ya-54\u001d\u0013¡",
            "sessionId": -17.0
        },
        {
            "questionerId": 16.0,
            "name": "U(\u0006?®-$ÇN¥¢Br*¸ufv¯'?\u0012[\u0018A",
            "sessionId": -11.0
        },
        {
            "questionerId": -16.0,
            "name": "W",
            "sessionId": -26.0
        },
        {
            "questionerId": 29.0,
            "name": null,
            "sessionId": -2.0
        },
        {
            "questionerId": -26.0,
            "name": "'j°]Ét\u0017\u0012./a>G\u0015",
            "sessionId": -12.0
        },
        {
            "questionerId": 22.0,
            "name": "xz\u001e",
            "sessionId": 30.0
        },
        {
            "questionerId": -25.0,
            "name": "&×ç4\u001ayöpÍ\u0004WÙ«\u001d#ÀDîP",
            "sessionId": -6.0
        },
        {
            "questionerId": 26.0,
            "name": "\u0012|Ém",
            "sessionId": -15.0
        },
        {
            "questionerId": 22.0,
            "name": "¢\u0004\u000eþ/yíå=ðk9j°\u0019>T7\\02",
            "sessionId": 18.0
        },
        {
            "questionerId": 18.0,
            "name": "#ÿ¥)H;ÙDPd \t]~",
            "sessionId": -6.0
        },
        {
            "questionerId": -15.0,
            "name": "Ò\"FD'\u001fNF§\u0019Û\u0012¨s\u0007g\u001c=Gñ\\ã&",
            "sessionId": -7.0
        },
        {
            "questionerId": 18.0,
            "name": "",
            "sessionId": -29.0
        },
        {
            "questionerId": 12.0,
            "name": "fË}p",
            "sessionId": -29.0
        },
        {
            "questionerId": 8.0,
            "name": " ",
            "sessionId": -9.0
        },
        {
            "questionerId": -5.0,
            "name": "H¬JH\u0005\r0\u0018",
            "sessionId": -19.0
        },
        {
            "questionerId": 10.0,
            "name": "\u0016\u0001¶l\u0014Ww:g7¨",
            "sessionId": 7.0
        },
        {
            "questionerId": 11.0,
            "name": "²1Q]ÙËeR\u0006A~ÚS \u0000/Òn[\u0001tQ(c\u0014",
            "sessionId": -7.0
        },
        {
            "questionerId": 7.0,
            "name": "]\u001f©£\u0017u#D\u001bZd«\"ß,p¼ÿ\u00135gSb\u0003U",
            "sessionId": 13.0
        },
        {
            "questionerId": -2.0,
            "name": "z\u0002ô\u000b\u000exgMn",
            "sessionId": 22.0
        },
        {
            "questionerId": 17.0,
            "name": null,
            "sessionId": 11.0
        },
        {
            "questionerId": -7.0,
            "name": "8\u0012aFP\u0002",
            "sessionId": -12.0
        },
        {
            "questionerId": 16.0,
            "name": null,
            "sessionId": -28.0
        },
        {
            "questionerId": -6.0,
            "name": "{eTg½]¼r\u001b\u001eX`gmyL",
            "sessionId": 23.0
        },
        {
            "questionerId": -19.0,
            "name": null,
            "sessionId": 19.0
        },
        {
            "questionerId": 10.0,
            "name": "&h!\u0018!a\r,@5\u000b6\u0003}p",
            "sessionId": 4.0
        },
        {
            "questionerId": -25.0,
            "name": null,
            "sessionId": 8.0
        },
        {
            "questionerId": -4.0,
            "name": "p\u0017Fbs\"%\u000c?ôOñºDåès",
            "sessionId": -28.0
        },
        {
            "questionerId": -11.0,
            "name": "\u00183¨_ù",
            "sessionId": 13.0
        },
        {
            "questionerId": -4.0,
            "name": "\u0010yQk\u000f]U\r.q",
            "sessionId": -19.0
        },
        {
            "questionerId": 9.0,
            "name": "",
            "sessionId": 25.0
        },
        {
            "questionerId": 23.0,
            "name": null,
            "sessionId": 9.0
        },
        {
            "questionerId": -19.0,
            "name": "Æ«\"u¥þ¼",
            "sessionId": 13.0
        },
        {
            "questionerId": -13.0,
            "name": "!<[\u0014úklkHöVÐD×x",
            "sessionId": -28.0
        },
        {
            "questionerId": -3.0,
            "name": ":n\u0005\u00059P)þY\u0001IBV",
            "sessionId": 19.0
        },
        {
            "questionerId": -1.0,
            "name": null,
            "sessionId": -23.0
        },
        {
            "questionerId": 28.0,
            "name": "\u0002#\"í`",
            "sessionId": -26.0
        },
        {
            "questionerId": -1.0,
            "name": "q0y¼® V<þ",
            "sessionId": -26.0
        },
        {
            "questionerId": 21.0,
            "name": "%",
            "sessionId": -2.0
        },
        {
            "questionerId": 24.0,
            "name": "\u001dû#t\"g¯HþhBÙ|àj-O#É°",
            "sessionId": -28.0
        },
        {
            "questionerId": -15.0,
            "name": "9·½\u000bq\u0012z_ßÐ\u001fH$AF",
            "sessionId": -11.0
        },
        {
            "questionerId": 8.0,
            "name": null,
            "sessionId": 29.0
        },
        {
            "questionerId": 30.0,
            "name": null,
            "sessionId": 29.0
        },
        {
            "questionerId": -22.0,
            "name": "F1%ñªí}Ìv\u000f\u001d\u0017",
            "sessionId": -17.0
        },
        {
            "questionerId": -27.0,
            "name": "`9DÝ3$Âý:µ}4ÅI`_xqÈ\\UW",
            "sessionId": 10.0
        },
        {
            "questionerId": -8.0,
            "name": null,
            "sessionId": 24.0
        },
        {
            "questionerId": -22.0,
            "name": null,
            "sessionId": -29.0
        },
        {
            "questionerId": 30.0,
            "name": "zE$\u0003«_\\E§}\u0010ë«y\u0006Ò/\u0008SNwy\u0001\tâ×¦",
            "sessionId": 22.0
        },
        {
            "questionerId": -7.0,
            "name": "e\u0010w\u001bzW\u001fBk\tm}+\u001f\u0011bÿ\u0011\u0006",
            "sessionId": 4.0
        },
        {
            "questionerId": -30.0,
            "name": "e¦S¬´uÙ}\u0008ç9¦À\u0016=|EhR®ÿRÙaõ",
            "sessionId": -16.0
        },
        {
            "questionerId": 28.0,
            "name": "\u0012\\nÞT\u0007\u0014ÀvNtV",
            "sessionId": 14.0
        },
        {
            "questionerId": 0.0,
            "name": "\u00039b)s>iCScxÂßL7±\u000c{\u00181Í",
            "sessionId": 2.0
        },
        {
            "questionerId": 10.0,
            "name": "\u0004",
            "sessionId": 2.0
        },
        {
            "questionerId": -18.0,
            "name": null,
            "sessionId": 30.0
        },
        {
            "questionerId": 1.0,
            "name": null,
            "sessionId": -7.0
        },
        {
            "questionerId": -10.0,
            "name": "Exì\taIa<S\u0010@Ws\u000fN¹0sî#s[\u0001ncü",
            "sessionId": -11.0
        },
        {
            "questionerId": 29.0,
            "name": "Z<`Ao\u000b\u0015\u0003>8jÛ}p\u0006¸",
            "sessionId": 7.0
        },
        {
            "questionerId": -9.0,
            "name": "0\u001dÿê\u001c\u001f\u0007!rbsbo",
            "sessionId": 14.0
        },
        {
            "questionerId": 10.0,
            "name": null,
            "sessionId": -15.0
        },
        {
            "questionerId": -26.0,
            "name": "/L:\u000b_8\u0017läÅuÿ\u0004c\u001b",
            "sessionId": -7.0
        },
        {
            "questionerId": -9.0,
            "name": null,
            "sessionId": 18.0
        },
        {
            "questionerId": -3.0,
            "name": null,
            "sessionId": -24.0
        },
        {
            "questionerId": -20.0,
            "name": "w",
            "sessionId": -22.0
        },
        {
            "questionerId": 17.0,
            "name": "gC@5>m\u0017ÈxÈ~\näc\u0011y\u0008\u0010=/",
            "sessionId": 0.0
        },
        {
            "questionerId": -30.0,
            "name": null,
            "sessionId": 25.0
        },
        {
            "questionerId": 6.0,
            "name": "Z\u0011õgCÜv",
            "sessionId": -29.0
        },
        {
            "questionerId": -30.0,
            "name": "\u0014\u000fJ",
            "sessionId": -6.0
        },
        {
            "questionerId": 14.0,
            "name": ",i?f\u0012pt\u0016+.",
            "sessionId": -27.0
        },
        {
            "questionerId": -19.0,
            "name": null,
            "sessionId": -23.0
        },
        {
            "questionerId": -19.0,
            "name": "",
            "sessionId": -27.0
        },
        {
            "questionerId": -20.0,
            "name": "5\u0015¿«^\u0015\n_\u00034b©q\u0004\u0013¿r",
            "sessionId": 21.0
        },
        {
            "questionerId": 19.0,
            "name": "f\u000f®*\u0014",
            "sessionId": -5.0
        },
        {
            "questionerId": 19.0,
            "name": "¥\u0019p\u001f Ø(T\u0005«²¤g¸_n¤",
            "sessionId": 19.0
        },
        {
            "questionerId": -3.0,
            "name": null,
            "sessionId": 4.0
        },
        {
            "questionerId": -18.0,
            "name": null,
            "sessionId": 15.0
        },
        {
            "questionerId": 13.0,
            "name": "ßkOùËÛ0Õ",
            "sessionId": 14.0
        },
        {
            "questionerId": -26.0,
            "name": null,
            "sessionId": 28.0
        },
        {
            "questionerId": 27.0,
            "name": "ßZbï+\nè}gn\u001e",
            "sessionId": -5.0
        },
        {
            "questionerId": -5.0,
            "name": "^&k*&j5DQ$DG,Qù\u0005$\\os\u0008kú",
            "sessionId": -12.0
        },
        {
            "questionerId": 10.0,
            "name": "",
            "sessionId": 29.0
        },
        {
            "questionerId": -26.0,
            "name": "\r]\u001b\u0015ÁNj\u0008áê~",
            "sessionId": 26.0
        },
        {
            "questionerId": 7.0,
            "name": "àmBø4DepzÊ\u001c\u0012ÌÉ\u001eÕpSæ",
            "sessionId": 8.0
        },
        {
            "questionerId": -28.0,
            "name": null,
            "sessionId": -3.0
        },
        {
            "questionerId": -7.0,
            "name": null,
            "sessionId": 13.0
        },
        {
            "questionerId": -16.0,
            "name": "ó§*³\u000fBÅ\u0004â`¡\u0005B#\tG",
            "sessionId": 24.0
        },
        {
            "questionerId": -13.0,
            "name": "WSMoèHì\u001dÜÁ",
            "sessionId": 2.0
        },
        {
            "questionerId": 22.0,
            "name": "D@L7Ê²³¶(3{¼\u0015WÐ\u0017%Ã",
            "sessionId": -20.0
        },
        {
            "questionerId": -4.0,
            "name": "\u0013",
            "sessionId": 2.0
        },
        {
            "questionerId": -25.0,
            "name": "¡3\u0019\u0008%Å\u00186\u000b7V?S\u001bt",
            "sessionId": 12.0
        },
        {
            "questionerId": 20.0,
            "name": "\u001dL0â¿$ÿF\u00116",
            "sessionId": 9.0
        },
        {
            "questionerId": 5.0,
            "name": "ëÇ9F!DtC\u001c+\u0010'z\u000b\u0004vxª­\u0005®ÑF8ò\u0012\n^Û",
            "sessionId": 17.0
        },
        {
            "questionerId": 29.0,
            "name": "\u0007õk=¸!<w°Bvµ",
            "sessionId": -21.0
        },
        {
            "questionerId": -19.0,
            "name": "",
            "sessionId": 8.0
        },
        {
            "questionerId": -22.0,
            "name": "j\u001c+*d(Cc?·ÁðN¬\u00155/F\u0007\u0017HsK\u001c\tJS",
            "sessionId": 16.0
        },
        {
            "questionerId": -15.0,
            "name": "@@\u00060\u00035VêÄ*\u0012Ê\u000b~J",
            "sessionId": 3.0
        },
        {
            "questionerId": 15.0,
            "name": "Çàj",
            "sessionId": -18.0
        },
        {
            "questionerId": 13.0,
            "name": "^¿\u0007J_\\\u000e\u0019Ô$bÁ\u0004Úa\u001c\";YeXGSË\u001cP=",
            "sessionId": -14.0
        },
        {
            "questionerId": -9.0,
            "name": "-é>1.·&2õAÉN¥`\u000f\u000f%§JT+1I",
            "sessionId": 12.0
        },
        {
            "questionerId": -14.0,
            "name": "¢O©F`A\u0002Jn¥\u001ea\u000c\u000cóAüõ]5\u001c",
            "sessionId": 27.0
        },
        {
            "questionerId": -26.0,
            "name": "ÐMkhûxg{'K",
            "sessionId": 12.0
        },
        {
            "questionerId": 2.0,
            "name": "ÓlI¥\u0010£j",
            "sessionId": -6.0
        },
        {
            "questionerId": 2.0,
            "name": null,
            "sessionId": 17.0
        },
        {
            "questionerId": -4.0,
            "name": null,
            "sessionId": -11.0
        },
        {
            "questionerId": -17.0,
            "name": null,
            "sessionId": -26.0
        },
        {
            "questionerId": -7.0,
            "name": null,
            "sessionId": -20.0
        },
        {
            "questionerId": -6.0,
            "name": "^<M-âÚÏßS\u0001¥w\u0013 ",
            "sessionId": 7.0
        },
        {
            "questionerId": -2.0,
            "name": "TT4p»\u001d5VÄ\u0006`h..YYs .r",
            "sessionId": -5.0
        },
        {
            "questionerId": -8.0,
            "name": "¼j³z¸\u0018\u00087",
            "sessionId": 9.0
        },
        {
            "questionerId": -14.0,
            "name": "Q7RgxJ;Qva\u00104z­0÷®úÛH",
            "sessionId": -25.0
        },
        {
            "questionerId": -18.0,
            "name": "\u000e(\u0012FZ\u0001O\u0000\u000c\u001d",
            "sessionId": -5.0
        },
        {
            "questionerId": 23.0,
            "name": "r=",
            "sessionId": -21.0
        },
        {
            "questionerId": -4.0,
            "name": null,
            "sessionId": -6.0
        },
        {
            "questionerId": 25.0,
            "name": null,
            "sessionId": -5.0
        },
        {
            "questionerId": 14.0,
            "name": "È$j\u0007þ'*>J+N:rMh\u0004ï+AI",
            "sessionId": -28.0
        },
        {
            "questionerId": 13.0,
            "name": "0\u0005gç\u0013}s':Ó\u0003èh3óxR",
            "sessionId": -10.0
        },
        {
            "questionerId": -27.0,
            "name": null,
            "sessionId": 22.0
        },
        {
            "questionerId": 16.0,
            "name": null,
            "sessionId": -20.0
        },
        {
            "questionerId": 18.0,
            "name": "\u0015^(4c¢",
            "sessionId": 30.0
        },
        {
            "questionerId": -16.0,
            "name": "/!J^;5ÝÞjfc?rÍÖy\u0012;\u000eÿH\u0016á",
            "sessionId": 29.0
        },
        {
            "questionerId": 9.0,
            "name": "ø\u0002âJ\u001b´õ\u001c^\u0002ì¬tÏhÓkkI4b",
            "sessionId": -25.0
        },
        {
            "questionerId": -18.0,
            "name": null,
            "sessionId": -16.0
        },
        {
            "questionerId": 1.0,
            "name": "\n\u0015Ú\u0019<y<Fa\u0011S",
            "sessionId": -12.0
        },
        {
            "questionerId": 11.0,
            "name": "\t\u001a4\u0019ÄmîZ³\u000fêx  ",
            "sessionId": 15.0
        },
        {
            "questionerId": -25.0,
            "name": "N$2_\u0011T#foc\u0010\rn+}",
            "sessionId": 10.0
        },
        {
            "questionerId": -30.0,
            "name": "ùVã_\u0000±'*¨öÝª$w*è=YV\u0015\u0016ÐYg\u001f/IÝ6",
            "sessionId": -21.0
        },
        {
            "questionerId": 21.0,
            "name": "Ù!{E$\u0006(ô\u0005|¼\u0008Â\u0006«û9\u0012Ùexÿ=\u0006",
            "sessionId": -11.0
        },
        {
            "questionerId": 15.0,
            "name": "ù\\)[\u001c\u001fXJ\u0019b}\u0002'q",
            "sessionId": -1.0
        },
        {
            "questionerId": -14.0,
            "name": "´L>ó",
            "sessionId": -15.0
        },
        {
            "questionerId": 27.0,
            "name": "W\u0018ý\u001b\u00152ÉS<Mi|\u0019/",
            "sessionId": -5.0
        },
        {
            "questionerId": -5.0,
            "name": "ùaes\u0006h(××:J{",
            "sessionId": 26.0
        },
        {
            "questionerId": 19.0,
            "name": null,
            "sessionId": -13.0
        },
        {
            "questionerId": -28.0,
            "name": "ô<\u001e\u000fDo#í,Ó\u0007«u}L<vj\u0003\\ó<'\u001e`",
            "sessionId": -20.0
        },
        {
            "questionerId": 7.0,
            "name": "B--\u0007ã3X!\u000fÖ''W¢A",
            "sessionId": 2.0
        }
    ]
}