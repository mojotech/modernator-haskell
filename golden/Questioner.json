{
    "seed": -2578633418132734637,
    "samples": [
        {
            "questionerId": -9,
            "name": "¥Å@b\u0005\u0016Q\u001e\u000e78+ò'd4:\u001d",
            "sessionId": -21
        },
        {
            "questionerId": -13,
            "name": "q\u001fC&E",
            "sessionId": -13
        },
        {
            "questionerId": 29,
            "name": "),õ",
            "sessionId": -24
        },
        {
            "questionerId": -9,
            "name": "yu¾\u0015)Áp~I\u0012c/",
            "sessionId": 5
        },
        {
            "questionerId": 28,
            "name": "îâÂ\u0005\\¨](E\u0004~¿\u0017\u0015RP î\u001c",
            "sessionId": -22
        }
    ]
}